osu.Game/Screens/Play/SkipOverlay.cs

```diff
@@ -39,6 +39,8 @@ public class SkipOverlay : VisibilityContainer, IKeyBindingHandler<GlobalAction>
         [Resolved]
         private GameplayClock gameplayClock { get; set; }
 
+        public bool Disabled { get; set; }
```

Comment:
This is not how this should be gone about. Internal use (inside of SkipOverlay) of alpha should be moved to a nested container.